config.h

```diff
@@ -12,6 +12,12 @@
 #define GC_USER ""
 #define GC_PASS ""
 
+// Comma-separated Prometheus labels (key=value) to apply to the metrics from the sensor
+// Example: site=home,location=balcony

```

Comment:
@wengelbrecht-grafana Re: providing an example, the library supports a few different styles for providing labels (though in practice it just ignores curly brackets and quotation marks)

```c
#define PROM_LABELS "site=home,location=balcony"
```

would add two labels for the Prometheus metric `m5stick_*{site="home", location="balcony"}`.
Do you think I should expand this to make it a bit clearer?

Reply:
This is perfect, thank you!